2gp-demo/force-app/main/default/classes/DebugFeatureAccessDemoController.cls

```diff
@@ -0,0 +1,14 @@
+public with sharing class DebugFeatureAccessDemoController {
```

Comment:
i'm thinking we could keep the debug classes and page out of the demo app, but keep it in a separate branch called "debug" or something so it's available

Reply:
Agree, will move these debug in internal git repo.